src/transformers/models/gptsan_japanese/tokenization_gptsan_japanese.py

```diff
@@ -258,16 +254,14 @@ def convert_tokens_to_string(self, tokens):
         text = "".join(words)
         return text
 
-    # Copied from tokenization_gpt_neox_japanese.GPTNeoXJapaneseTokenizer._build_conversation_input_ids
-    def _build_conversation_input_ids(self, conversation: "Conversation") -> List[int]:
-        """This corresponds to DialoGPT variants of models."""
-        input_ids = []
-        for is_user, text in conversation.iter_texts():
-            input_ids.extend(self.encode(text, add_special_tokens=False) + [self.eos_token_id])
-
-        if len(input_ids) > self.model_max_length:
-            input_ids = input_ids[-self.model_max_length :]
-        return input_ids
+    @property
```

Comment:
Is the removal of the `# Copied from` voluntary?

Reply:
Flagging as it's not back yet!